File: static/include/examples-generated/plan-motion-arm-gripper-samples.snippet.move-to-pose.go
// Generate a sample "start" pose to demonstrate motion
testStartPose := spatialmath.NewPose(
	r3.Vector{X: 510.0, Y: 0.0, Z: 526.0},
	&spatialmath.OrientationVectorDegrees{OX: 0.7071, OY: 0.0, OZ: -0.7071, Theta: 0.0},
)
testStartPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, testStartPose)

moveReq := motion.MoveReq{
	ComponentName: myArmResourceName,
	Destination:   testStartPoseInFrame,
	WorldState:    worldState,
}
_, err = motionService.Move(context.Background(), moveReq)
if err != nil {
	logger.Fatal(err)
}

File: static/include/examples-generated/plan-motion-arm-gripper-samples.snippet.move-to-pose.py
# Generate a sample "start" pose to demonstrate motion
test_start_pose = Pose(x=510.0,
                       y=0.0,
                       z=526.0,
                       o_x=0.7071,
                       o_y=0.0,
                       o_z=-0.7071,
                       theta=0.0)
test_start_pose_in_frame = PoseInFrame(reference_frame="world",
                                       pose=test_start_pose)

await motion_service.move(component_name=ARM_NAME,
                          destination=test_start_pose_in_frame,
                          world_state=world_state)
